news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.
+This entire mapset holds quite its value in there and we hope all of you can get a pleasant time while trying to get some points out of this set!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/sound-chimera.jpg)](http://osu.ppy.sh/beatmapsets/779916)
+
+[yea](https://osu.ppy.sh/users/6874332)’s map of Laur's “Sound Chimera” comes in a close 3rd place for this quarter’s spotlights featuring a ton of finishers and some groovy slider velocity usage! Being the first marathon out of many for this round of spotlights, it certainly stands out in its own unique way. This map is truly a finished product, with Laur’s signature diversity in music and yea’s impeccable translation of that through intuitive finisher usage, slider velocity and density balance.
+
+yea is relatively new to ranking maps, but currently has 5 loved maps ranging from 7 to 9 stars, so if you enjoy a challenge, head over to his profile. This truly is just a fun map, and I am not doing it any justice trying to sit here explaining why it’s so good, so just go and give it a play to find out for yourself!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/last-minute.jpg)](http://osu.ppy.sh/beatmapsets/835011)
+
+A pp map in spotlights? No way!
+
+[\[Zeth\]](https://osu.ppy.sh/users/9912966)’s map of sanj's “last minute (sakuraburst remix)” breaks the mold with subtle but tasteful sv, a complex and interesting song, 1/8 doubles like you’ve never seen and pp for only deserving parties. Marathons maps often stand out due to their ability to change pace and texture several times during one map, and boy, does \[Zeth] sure capitalize on that.
+
+The use of 1/8 doubles in this map is what makes it stand out among other similar marathons, since these make sense in the context of the music. With the 1/6 in the kiais leading up to them, they make for a great intensity boost, and are prolonged for such a time that separates the goods from the greats when it comes down to getting a full combo.
+
+With this being \[Zeth]’s first and only ranked map so far, only time can tell what will come of this talented individual.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/egret.jpg)](http://osu.ppy.sh/beatmapsets/821745)
+
+Holding quite the catalogue of ranked maps in the Spotlights section, [Nifty](https://osu.ppy.sh/users/4956097) expands on it with another beautiful mapset, this time with the song Dirty Androids' "Egret and Willow", and boy is this a good one!
```

Comment:
"with the song "Egret and Willow" from Dirty Androids" sounds much better

Reply:
Removed "the song" instead. Prefer to have consistency on artist and title formatting